modules/menu/menu.c

```diff
@@ -308,6 +308,9 @@ static int cmd_answer(struct re_printf *pf, void *unused)
 	int err;
 	(void)unused;
 
+	if (!ua_call(ua))
+		return 0;
+
```

Comment:
this will return any call, even in state outgoing or established.

I would prefer that you lookup the first call that is in the state `STATE_INCOMING`
and then answer this call. if there are no calls in the state INCOMING it should print
something like "no pending incoming call to answer"

Reply:
> or established.

This function is not accessible if call in `ESTABLISHED` state, isnt?

Reply:
I was guided by the logic from `ua_hold_answer()`: https://github.com/alfredh/baresip/blob/59c9beb41400ee9e8a180af5a32e42ae281ba6d7/src/ua.c#L975-L992